shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -282,17 +284,12 @@ - (void)performCommonViewControllerInitialization {
   }
 
   _initialized = YES;
-
   _orientationPreferences = UIInterfaceOrientationMaskAll;
   _statusBarStyle = UIStatusBarStyleDefault;
 
   [self setUpNotificationCenterObservers];
 }
 
-- (FlutterEngine*)engine {
-  return _engine.get();
-}
-
 - (fml::WeakNSObject<FlutterViewController>)getWeakNSObject {
   return _weakFactory->GetWeakNSObject();
 }
```

Comment:
Still needed by FlutterEngine, which calls this, and puts it into a field in a C++ class in platform_view_ios.h, which I'll migrate in the next patch.